partiql-lang/src/main/kotlin/org/partiql/lang/prettyprint/QueryPrettyPrinter.kt

```diff
@@ -320,7 +320,14 @@ class QueryPrettyPrinter {
             is PartiqlAst.Expr.Or -> writeNAryOperator("OR", node.operands, sb, level)
             is PartiqlAst.Expr.InCollection -> writeNAryOperator("IN", node.operands, sb, level)
             is PartiqlAst.Expr.BagOp -> {
-                var name = node.op.javaClass.simpleName.toUpperCase().replace("_", " ")
+                var name = when (node.op) {
```

Comment:
Buggy previous behavior would print `OUTER UNION` node as `OUTERUNION`.

Could have been more clever w/ printing but since this is legacy code that will be removed ahead of `v1`, just opted for a simple solution.